content/docs/welcome-guide/setup/setup-from-installer.md

```diff
@@ -2,8 +2,50 @@
 title: Setting up O3DE from the Installer
 description: Learn how to install and set up Open 3D Engine (O3DE) using the installer tool.
 weight: 200
+draft: true
 ---
 
-{{< placeholder >}}
+A quick way to get started with O3DE is to download and install a released version. After running the installer, you will have a stable, pre-built version of the engine and its Gems, ready for you to jump into your next project.
 
-Coming soon! For now, follow the instructions for installing O3DE from GitHub found in [Setup O3DE from GitHub](./setup-from-github).
+The instructions here will guide you through the installation experience. At the end of a successful install, you will be ready to create new or open existing projects with the **Project Manager** tool.
+
+## Prerequisites
+
+The instructions here assume you have completed the following:
+
+* Met all hardware and software requirements listed in [System Requirements](./requirements).
+* Configured the required software as described in [Software Configuration](./requirements#software-configuration).
+
+## Installing O3DE
+
+1. Obtain the latest version of the installer from the [O3DE download](https://o3de.org/download/) page.
+
+1. Run the installer from your download location.
+
+1. During developer preview, you might encounter a warning such as the following from Microsoft Defender describing the program as an unrecognized app. To proceed with installation, you must first open the **More info** link and then choose **Run anyway**.
```

Comment:
```suggestion
1. During the developer preview, you might encounter the following warning from Microsoft Defender describing the program as an unrecognized app. To proceed with installation, choose **More info**, and then choose **Run anyway**.
```
The current wording describes a warning "such as" this Microsoft Defender sideloading warning. Might the reader receive a similar warning? If not, see my suggested changes above. If so, I recommend something like the following wording instead for clarity:
>During the developer preview, you might encounter a warning about sideloading the app, such as the following warning from Microsoft Defender. If you get this particular warning, to proceed with installation, choose **More info**, and then choose **Run anyway**.

Reply:
When I wrote this, I wasn't sure if we wanted to allude to scenarios where a reader is using something besides Defender, and therefore might encounter a different warning. In retrospect, I think this unnecessarily complicates this text, which is only temporary during the developer preview anyway. I'll go with your first suggestion.